src/bin/copy_builder_test_machine.js

```diff
@@ -0,0 +1,21 @@
+import { testerConfig, testerOptions } from '../machines/builder_route/tester.config';
```

Comment:
I'd add a `copy_all` script to copy a complete machine with testers to ease porting to the xstate visualizer